src/status_im/transport/message/core.cljs

```diff
@@ -0,0 +1,6 @@
+(ns status-im.transport.message.core)
+
+(defprotocol StatusMessage
+  "Protocol for transport layed status messages"
```

Comment:
Typo and maybe elaborate a bit. Maybe mini example usage of how to mock it? Right now it's stuff like "chat-id" "signature" and "cofx" - things that require domain expertise to understand.